boards/arm/nucleo_h723zg/Kconfig.defconfig

```diff
@@ -0,0 +1,42 @@
+# STM32H723ZG Nucleo board configuration
+
+# Copyright (c) 2020 Alexander Kozhinov
+# AlexanderKozhinov@yandex.com
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NUCLEO_H723ZG
+
+config BOARD
+	default "nucleo_h723zg"
+
+config CLOCK_STM32_D1CPRE
+	default 1
+
+config CLOCK_STM32_HPRE
+	# HCLK: 275MHz
+	default 2
+
+config CLOCK_STM32_D2PPRE1
+	# APB1: 137.5MHz
+	default 2
+
+config CLOCK_STM32_D2PPRE2
+	# APB2: 137.5MHz
+	default 2
+
+config CLOCK_STM32_D1PPRE
+	# APB3: 137.5MHz
+	default 2
+
+config CLOCK_STM32_D3PPRE
+	# APB4: 137.5MHz
+	default 2
```

Comment:
This way of configuring clock was proposed on dual core boards for dual core specific reasons, as it allows to share a common clock config between targets (avoiding potential desync).
For single core targets, we usualy put these in _defconfig file, as for instance:
```
CONFIG_CLOCK_STM32_PLL_M_DIVISOR=16
CONFIG_CLOCK_STM32_PLL_N_MULTIPLIER=336
CONFIG_CLOCK_STM32_PLL_P_DIVISOR=4
CONFIG_CLOCK_STM32_PLL_Q_DIVISOR=7
CONFIG_CLOCK_STM32_AHB_PRESCALER=1
CONFIG_CLOCK_STM32_APB1_PRESCALER=2
CONFIG_CLOCK_STM32_APB2_PRESCALER=1
```

Reply:
done